docker-compose.yml

```diff
@@ -0,0 +1,136 @@
+version: '3'
+networks:
+  default:
+    driver: bridge
+services:
+  libpostal:
+    image: pelias/libpostal-service
+    container_name: pelias_libpostal
+    user: "${DOCKER_USER}"
+    restart: always
+    ports: [ "127.0.0.1:${PELIAS_LIBPOSTAL_PORT:-4400}:4400" ]
```

Comment:
Since libpostal won't provide a custom port configuration inside the container it stays on 4400 internally.